DepositeCalcTests/Tests/CalculatorPageTests.cs

```diff
@@ -0,0 +1,230 @@
+﻿using DepositeCalcTests.Pages;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using System.Globalization;
+using System.Text.RegularExpressions;
+
+namespace DepositeCalcTests.Tests
+{
+    internal class CalculatorPageTests
+    {
+        private IWebDriver driver;
+
+        [SetUp]
+        public void Setup()
+        {
+            ChromeOptions options = new ChromeOptions { AcceptInsecureCertificates = true };
+            driver = new ChromeDriver(options);
+            driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(30);
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(5);
+            driver.Url = "https://localhost:5001/Calculator";
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            driver.Quit();
+        }
+
+        [TestCase("", "1", "1")]
+        [TestCase("1", "", "1")]
+        [TestCase("1", "1", "")]
+        public void MandaroryTextFieldsWith365BtnTest(string depositAmount, string interestRate, string investmentTerm)
+        {
+            // Arrange
+            var calculatorPage = new CalculatorPage(driver);
+
+            // Act
+            calculatorPage.MandatoryTextFields(depositAmount, interestRate, investmentTerm);
+            calculatorPage.FinancialYear = "365";
+
+            // Assert
+            Assert.IsTrue(calculatorPage.IsCalculateBtnDisabled, "Calculate button is clickable. This means that one of the text fields or one of the Financial year radio buttons are not mandatory");
+        }
+
+        [TestCase("", "1", "1")]
+        [TestCase("1", "", "1")]
+        [TestCase("1", "1", "")]
+        public void MandaroryTextFieldsWith360BtnTest(string depositAmount, string interestRate, string investmentTerm)
+        {
+            // Arrange
+            var calculatorPage = new CalculatorPage(driver);
+
+            // Act
+            calculatorPage.MandatoryTextFields(depositAmount, interestRate, investmentTerm);
+            calculatorPage.FinancialYear = "360";
+
+            // Assert
+            Assert.IsTrue(calculatorPage.IsCalculateBtnDisabled, "Calculate button is clickable. This means that one of the text fields or one of the Financial year radio buttons are not mandatory");
+        }
+
+        [TestCase(1, 1, 1, "1.00", "0.00")]
+        [TestCase(10000, 100, 365, "20,000.00", "10,000.00")]
+        public void ValidCalculation365Test(double depositAmount, double interestRate, double investmentTerm, string expectedIncome, string expectedInterestEarned)
```

Comment:
1. You can merge these two tests by adding `finYear` parameter.
2. You don't need these parameters to have type `double`. 

```suggestion
        [TestCase("360", "10000", "100", "360", "20,000.00", "10,000.00")]
        [TestCase("365", "10000", "100", "365", "20,000.00", "10,000.00")]
        public void ValidCalculationTest(string finYear, string depositAmount, string interestRate, string investmentTerm, string expectedIncome, string expectedInterestEarned)
```